.github/workflows/build-test-publish-images.yml

```diff
@@ -191,10 +191,11 @@ jobs:
         py${{ matrix.PYTHON_VER }}-\
         ${{ matrix.ARCH }}"
   build-multiarch-manifest:
-    if: inputs.build_type == 'branch'
```

Comment:
Will make this `build-multiarch-manifest` job `branch` only after this PR is reviewed.